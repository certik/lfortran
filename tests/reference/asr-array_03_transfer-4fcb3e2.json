{
    "basename": "asr-array_03_transfer-4fcb3e2",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/array_03_transfer.f90",
    "infile_hash": "d554a75d7567a6aed407dae5c430c9dea3fa17c13fad6b7ffeecb8d0",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-array_03_transfer-4fcb3e2.stdout",
    "stdout_hash": "5927c4bf6679eb3d4c63faa14992ac89b087b11c65c6986c03ef9bf7",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}