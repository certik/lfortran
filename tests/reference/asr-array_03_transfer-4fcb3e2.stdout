(TranslationUnit (SymbolTable 1 {array_03_transfer: (Program (SymbolTable 2 {x: (Variable 2 x Local (IntegerConstant 21432 (Integer 4 [])) () Save (Integer 4 []) Source Public Required .false.)}) array_03_transfer [] [(Print () [(BitCast (Var 2 x) (RealConstant   1.00000000000000000e+00 (Real 4 [])) () (Real 4 []) ())] () ())])}) [])
